### PR TITLE
Copy go/v1beta1 files to go/v1

### PR DESCRIPTION
**Pull request by noahklein**

### CODE REVIEW

File: go/v1/api/grafeas.go [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package grafeas is an implementation of the v1beta1 https://github.com/grafeas/grafeas/ API.
+package grafeas
+
+import (
+	"github.com/grafeas/grafeas/go/errors"
+	"github.com/grafeas/grafeas/go/iam"
+	gpb "github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto"
```
Comment (aysylu) [inline go/v1/api/grafeas.go]: This should be v1, right?

File: go/v1/api/grafeas.go [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+	PrepareCtx(ctx context.Context, projectID string) context.Context
+	Info(ctx context.Context, args ...interface{})
+	Infof(ctx context.Context, format string, args ...interface{})
+	Warning(ctx context.Context, args ...interface{})
+	Warningf(ctx context.Context, format string, args ...interface{})
+	Error(ctx context.Context, args ...interface{})
+	Errorf(ctx context.Context, format string, args ...interface{})
+}
+
+// API implements the methods in the v1beta1 Grafeas API.
```
Comment (aysylu) [inline go/v1/api/grafeas.go]: s/v1beta1/v1 everywhere appropriate please

File: go/v1/api/validators/vulnerability/vulnerability_test.go [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vulnerability
+
+import (
+	"testing"
+
+	pkgpb "github.com/grafeas/grafeas/proto/v1beta1/package_go_proto"
+	vulnpb "github.com/grafeas/grafeas/proto/v1beta1/vulnerability_go_proto"
```
Comment (aysylu) [inline go/v1/api/validators/vulnerability/vulnerability_test.go]: I think you'll need to generate these as well.

**Comment by noahklein** [comment]
* Updated Makefile to build the v1 go proto libraries.
* Built the v1 go proto libraries.
* Updated V1 imports to point to v1 dependencies.

@aysylu, @vtsao PTAL

File: Makefile [hunk 4]
```diff
@@ -45,57 +47,20 @@ grafeas_go_v1alpha1: .install.protoc-gen-go .install.grpc-gateway v1alpha1/proto
 		--swagger_out=logtostderr=true:. \
 		v1alpha1/proto/grafeas.proto
 
-define gen_go_proto
-	protoc/bin/protoc \
-		-I ./ \
+%_go_proto: %.proto protoc/bin/protoc install.tools
```
Comment (aysylu) [inline Makefile]: Maybe add a high-level comment above for what this does?

File: go/v1/api/validators/README.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# Grafeas API validators
+
+When making changes to these validators, please ensure to update the
+documentation on the protos so they match the code. E.g., if what fields are
+required change for a note resource, please update the note message proto's
```
Comment (aysylu) [inline go/v1/api/validators/README.md]: I'm not sure how to parse this part, could you please rephrase?

File: proto/v1/swagger/image.swagger.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (aysylu) [inline proto/v1/swagger/image.swagger.json]: Let's remove all the swagger specs for now, as we might not be generating client libraries from Swagger spec going forward.

File: proto/v1beta1/swagger/grafeas.swagger.json [hunk 7]
```diff
@@ -15,13 +15,13 @@
     "application/json"
   ],
   "paths": {
-    "/v1beta1/{name}": {
+    "/v1beta1/{name=projects/*/notes/*}": {
```
Comment (aysylu) [inline proto/v1beta1/swagger/grafeas.swagger.json]: Was this auto-generated or manually added? I don't believe this path would actually be interpreted correctly by swagger because OpenAPI doesn't support paths with wildcards in them.

File: proto/v1beta1/swagger/grafeas.swagger.json [hunk 8]
```diff
@@ -15,13 +15,13 @@
...
       "get": {
         "summary": "Gets the specified note.",
         "operationId": "GetNote",
         "responses": {
           "200": {
-            "description": "",
+            "description": "A successful response.",
```
Comment (aysylu) [inline proto/v1beta1/swagger/grafeas.swagger.json]: Revert this change and the ones below? This should be unchanged, right?

File: proto/v1beta1/swagger/project.swagger.json [hunk 9]
```diff
@@ -21,7 +21,7 @@
         "operationId": "ListProjects",
         "responses": {
           "200": {
-            "description": "",
+            "description": "A successful response.",
```
Comment (aysylu) [inline proto/v1beta1/swagger/project.swagger.json]: This file should be unchanged by the PR, right? If so, let's revert all the previous changes.

File: v1alpha1/proto/grafeas.swagger.json [hunk 10]
```diff
@@ -17,11 +17,11 @@
   "paths": {
     "/v1alpha1/projects": {
       "get": {
-        "summary": "Deletes the given `Occurrence` from the system. Use this when\nan `Occurrence` is no longer applicable for the given resource.",
+        "summary": "Lists projects.",
```
Comment (aysylu) [inline v1alpha1/proto/grafeas.swagger.json]: Does this PR affect this file? If not, let's revert the changes here.

File: proto/v1beta1/swagger/grafeas.swagger.json (hunk 7 above)
Comment (vtsao) [reply proto/v1beta1/swagger/grafeas.swagger.json]: These are likely caused by a new version of protoc that we've pulled in because I don't think we lock in a specific version of protoc to use. So these changes should be fine.
